src/seedu/addressbook/data/person/Block.java

```diff
@@ -0,0 +1,20 @@
+package seedu.addressbook.data.person;
+
+
+public class Block {
+	
+	public static final String EXAMPLE = "123";
+	private int blockNum;	
+	
+	// No validation check need as constructor only accepts integer as block number
+	public Block(int blockNumber) {
+		this.blockNum = blockNumber;
```

Comment:
Indentation problem here and few more places in the code?. Our coding standard requires you to use 4 spaces instead of tabs. You can configure Eclipse to convert tabs to spaces automatically.